user-api/src/main/java/com/example/auction/user/api/User.java

```diff
@@ -13,12 +13,14 @@
     private final UUID id;
     private final String name;
     private final String email;
+    private final String passwordHash;
 
     @JsonCreator
-    public User(@JsonProperty("id") UUID id, @JsonProperty("name") String name, @JsonProperty("email") String email) {
+    public User(@JsonProperty("id") UUID id, @JsonProperty("name") String name, @JsonProperty("email") String email ,@JsonProperty("password") String passwordHash) {
```

Comment:
Just a reminder from https://github.com/lagom/online-auction-java/pull/99#pullrequestreview-45036543: we _do not_ want to include the password hash in `User`, as it will then be returned in `getUser` and `getUsers` calls. We don't want to expose this information. It should stay internal to the `UserService`.